lib/matplotlib/lines.py

```diff
@@ -467,11 +467,12 @@ def contains(self, mouseevent):
         yt = xy[:, 1]
 
         # Convert pick radius from points to pixels
-        if self.figure is None:
+        fig = self.get_figure(root=False)
```

Comment:
```suggestion
        fig = self.get_figure(root=True)
```

for the same reason as the rest of the DPI related comments.